package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A standalone version of the readability library used for Firefox Reader View.",
   "main": "Readability.js",
   "scripts": {
-    "test": "mocha test/index.js",
+    "test": "mocha test/index.js test/test-jsdomparser.js",
```

Comment:
How about `mocha test/*_test.js` and append `_test.js` to the filename of whatever is a test file? That would ease adding more test files in the future.